http/task_service.go

```diff
@@ -334,12 +334,18 @@ func decodeGetTasksRequest(ctx context.Context, r *http.Request) (*getTasksReque
 		req.filter.After = id
 	}
 
-	if orgID := qp.Get("organization"); orgID != "" {
-		id, err := platform.IDFromString(orgID)
+	if orgName := qp.Get("org"); orgName != "" {
+		o, err := orgs.FindOrganization(ctx, platform.OrganizationFilter{Name: &orgName})
```

Comment:
I think there needs to be some way to obfuscate the error returned here, so, it is not possible to fish for organization names.

Perhaps the API should return an empty list of tasks of the org name is unknown?

Reply:
I think in other places (possibly only in some wip), I've used the phrase "unauthorized or unknown" to keep it vague.